lib/order.rb

```diff
@@ -0,0 +1,14 @@
+require './lib/food_list.rb'
+require 'money'
+
+class Order
+  include FoodList
+
+  def show_total
+    Money.new(items.inject(0) { |sum, item| sum + item.price }, "GBP").format
```

Comment:
To be a little more strictly `Single Responsibility` this logic:
```items.inject(0) { |sum, item| sum + item.price }``` could be separated into its own function to test the logic independently of presenting this.